src/devcards.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DevCards from './components/DevCards.jsx';
+
+/**
+ * Cards
+ */
+
+// Main card
+export default function devcard(name, doc, body) {
+  devcard.current.push({name, doc, body});
+}
+
+// Noop card
+devcard.off = Function.prototype;
```

Comment:
I've not seen this approach for no-op before, what's the advantage over `function() {}` ?